droopler.install

```diff
@@ -521,3 +521,54 @@ function droopler_update_8103() {
 
   $updater->checklist()->saveProgress($progress);
 }
+
+/**
+ * Droopler 2.1 Updates.
+ */
+function droopler_update_8104() {
+  /** @var Drupal\d_update\Updater $updater */
+  $updater = \Drupal::service('d_update');
+
+  // Add and update configs for media.
+  $configs = [
```

Comment:
Ten hook tak nie zadziała, w ogóle nie wrzucałbym go w tym tasku.
W ten sposób można wciągnąć tylko configi, których dotychczas nie było w Drooplerze.
Jeśli config już istnieje to ta tablica powinna zawierać jego hash z wersji 1.4, wygenerowany przez `drush gch <nazwa configu>`

Reply:
Wiem, sam to robiłem, ale czy bez hasha on po prostu nie wymusi importu tego configu i tak? Jezeli ktos nadpisal sobie te ustawienia i config sie nie wciagnie to w sumie całe media nie będzie działać i tak, więc trochę pomyslalem ze to trzeba na userach wymusic.

Reply:
Moemy ew. zostawić teraz hook a potej ja go zintegruje z poprzednim